src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java

```diff
@@ -179,6 +180,17 @@ private void sendResponseToApplicationEmails(GenericTseApplicationType applicati
             caseId,
             application.getType()
         );
+
+        notificationService.sendResponseEmailToClaimant(
```

Comment:
Would it worth adding a unit test to verify sendResponseEmailToClaimant is called?

Reply:
Added.